### ISSUE TITLE
Error: Verify that the package.json has a valid "main" entry

### ISSUE THREAD

**Issue by OllieJT** [akaSybe/rollup-plugin-svelte-ssr#8]
Most of the issues I come across are a mistake of my own, but I can't seem to figure out what it is this time. All errors seem to point with an issue in the package.json for the plugin.

**Steps to reproduce:**
1. Scaffold [simple starter](https://github.com/sveltejs/template) following instructions.
2. Modify rollup config for `rollup-plugin-svelte-ssr` (pasted below)
3. Build or Watch

**Error**
```
[!] Error: Cannot find module '/Users/<USERNAME>/Documents/GitHub/svelte-app/node_modules/rollup-plugin-svelte-ssr/dist/index.js'. Please verify that the package.json has a valid "main" entry
```

**Rollup Config**
```javascript
import svelte from "rollup-plugin-svelte";
import resolve from "@rollup/plugin-node-resolve";
import commonjs from "@rollup/plugin-commonjs";
import livereload from "rollup-plugin-livereload";
import { terser } from "rollup-plugin-terser";
import ssr from "rollup-plugin-svelte-ssr";

const production = !process.env.ROLLUP_WATCH;

export default {
	input: "src/App.svelte",
	output: {
		format: "cjs",
		file: "dist/ssr.js",
	},
	plugins: [
		svelte({
			generate: "ssr",
			// enable run-time checks when not in production
			dev: !production,
			// we'll extract any component CSS out into
			// a separate file - better for performance
			css: (css) => {
				css.write("public/build/bundle.css");
			},
		}),

		// If you have external dependencies installed from
		// npm, you'll most likely need these plugins. In
		// some cases you'll need additional configuration -
		// consult the documentation for details:
		// https://github.com/rollup/plugins/tree/master/packages/commonjs
		resolve({
			browser: true,
			dedupe: ["svelte"],
		}),
		commonjs(),

		// In dev mode, call `npm run start` once
		// the bundle has been generated
		!production && serve(),

		// Watch the `public` directory and refresh the
		// browser on changes when not in production
		!production && livereload("public"),

		// If we're building for production (npm run build
		// instead of npm run dev), minify
		production && terser(),

		ssr({
			fileName: "ssr.html",
			props: {
				name: "Hello",
			},
		}),
	],
	watch: {
		clearScreen: false,
	},
};

function serve() {
	let started = false;

	return {
		writeBundle() {
			if (!started) {
				started = true;

				require("child_process").spawn(
					"npm",
					["run", "start", "--", "--dev"],
					{
						stdio: ["ignore", "inherit", "inherit"],
						shell: true,
					},
				);
			}
		},
	};
}
```

**Comment by akaSybe** [comment]
Hi, @OllieJT! 

Thank you for report, I'll take a look at this in a few days.

**Comment by akaSybe** [comment]
@OllieJT I'm sorry, it was error in Travis CI config.

error fixed, new version 1.0.2 is available now

**Comment by OllieJT** [comment]
Thank you! Will test later.
